config/hubs/2i2c.cluster.yaml

```diff
@@ -48,10 +48,10 @@ hubs:
           config:
             Authenticator:
               allowed_users: &staging_users
-                - yuvipanda@gmail.com
+                # this will be replaced with the actual list of
+                # staff google ids by the deployer script
+                - <staff_google_ids>
```

Comment:
@choldgraf, what do you think about this? (I kept the `staff` prefix here, otherwise `google_ids` would've lost its explicitly )